app/src/main/java/com/nervousfish/nervousfish/modules/pairing/AndroidBluetoothHandler.java

```diff
@@ -12,7 +12,8 @@
 import java.io.ObjectOutputStream;
 
 /**
- * Helper class for {@link AndroidBluetoothService}
+ * Helper class for {@link AndroidBluetoothService} and acts as the bridge between the client and
+ * the service.It simplifies the whole Bluetooth functionality to the methods start, connect, stop and send
```

Comment:
Why between the client and the service? Isn't the service also responsible for client functionality. Also could you please put a space afte the dot and add a dot at the end of the sentence?